tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationControllerBuilder.cs

```diff
@@ -48,15 +53,32 @@ internal OrganisationControllerBuilder WithUpdateOrganisation(Organisation organ
             return this;
         }
 
+        internal OrganisationControllerBuilder WithCreateOrganisation()
+        {
+            var repositoryMock = new Mock<IOrganisationRepository>();
+            repositoryMock.Setup(x => x.CreateOrganisationAsync(It.IsAny<Organisation>()));
+            _organisationRepository = repositoryMock.Object;
+            return this;
+        }
```

Comment:
Is this ever used/required without `WithCreateOrganisationService`?

Reply:
Not really.

Would you suggest refactoring the code in this method to be part of `WithCreateOrganisationService`  (soon to be renamed) instead?

Reply:
I think I'd be tempted to do that or make it a private method that you invoke from the renamed `WithCreateOrganisationService`.

Reply:
👍